src/autocapture-utils.ts

```diff
@@ -483,7 +483,7 @@ function elementsToString(elements: PHElement[]): string {
                 ([key, value]) => (sortedAttributes[escapeQuotes(key.toString())] = escapeQuotes(value.toString()))
             )
         el_string += ':'
-        el_string += entries(attributes)
+        el_string += Object.entries(sortedAttributes)
```

Comment:
This is the actual change. The `sortedAttributes` was created above but not used here, so I guess this was what was initially intended somehow.